services/src/main/java/org/keycloak/organization/admin/resource/OrganizationsResource.java

```diff
@@ -186,6 +186,26 @@ public OrganizationResource get(@PathParam("org-id") String orgId) {
         return new OrganizationResource(session, organizationModel, adminEvent);
     }
 
+    /**
+     * Returns the organizations counts.
+     *
+     * @return
+     */
+    @GET
+    @NoCache
+    @Path("count")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Tag(name = KeycloakOpenAPI.Admin.Tags.ORGANIZATIONS)
+    @Operation(summary = "Returns the organizations counts.")
+    public Map<String, Long> getOrganizationCount(@QueryParam("search") String search) {
+        auth.realm().requireManageRealm();
+        Organizations.checkEnabled(provider);
+
+        Long count = provider.getAllStream(search, false, null, null).count();
+
+        return Map.of("count", count);
```

Comment:
This is in-line with the groups count with returns a map. I see that the users API returns an integer directly. Not sure what is the preferred option here.

Reply:
@pedroigor any preferences? I don't know what is the benefit of returning a map here, so I would personally return an `Integer` like we do in the users endpoint.

Reply:
Me neither. I share the same thoughts/solution.